.github/workflows/check-complete.yml

```diff
@@ -28,5 +21,17 @@ jobs:
         with:
           fetch-depth: 1
 
-      - name: Check all files
-        run: bash ./check.sh
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Install test dependencies
```

Comment:
Want to extend with caching? 

https://github.com/actions/cache/blob/main/examples.md#python---pip